### ISSUE TITLE
Travis CI - The command "script/cibuild" exited with 1.

### ISSUE THREAD

**Issue by suriyaa** [bunto/bunto-sitemap#1]
Errors founded here:

``` bash
Finished in 2.01 seconds (files took 1.13 seconds to load)
25 examples, 4 failures

Failed examples:

rspec ./spec/bunto-sitemap_spec.rb:76 # Bunto::BuntoSitemap generates the correct date for each of the posts
rspec ./spec/bunto-sitemap_spec.rb:48 # Bunto::BuntoSitemap puts all the posts in the sitemap.xml file
rspec ./spec/bunto-sitemap_spec.rb:142 # Bunto::BuntoSitemap with site url that needs URI encoding performs URI encoding of site url
rspec ./spec/bunto-sitemap_spec.rb:130 # Bunto::BuntoSitemap with a baseurl correctly adds the baseurl to the posts

Randomized with seed 20042

The command "script/cibuild" exited with 1.
Done. Your build exited with 1.
```

> See https://travis-ci.org/bunto/bunto-sitemap/jobs/118505864!

**Comment by suriyaa** [comment]
> Fixed in:
> - :white_check_mark: [**`692a10b`**](https://github.com/bunto/bunto-sitemap/commit/692a10ba38dd8ac3c913d3fe1df0c87c382b659d) commit
> - :white_check_mark: [**`c70a122`**](https://github.com/bunto/bunto-sitemap/commit/c70a122da301ec41c60f3df2966e8f9b6a57073c) commit
> - :white_check_mark: [**`27224af`**](https://github.com/bunto/bunto-sitemap/commit/27224afa80ba238f69bbeb562e7de98360097917) commit
> - :white_check_mark: [**`2a990b1`**](https://github.com/bunto/bunto-sitemap/commit/2a990b159416a4f0c7b8a12ab97face5e7386832) commit
> - :white_check_mark: [**`c372d4a`**](https://github.com/bunto/bunto-sitemap/commit/c372d4a7380a7339a5fe874d928691020d716409) commit

---

New failures:

``` bash
Failed examples:

rspec ./spec/test_bunto-last-modified-at.rb:26 # Bunto::BuntoSitemap with bunto-last-modified-at correctly adds the modified time to the posts
rspec ./spec/test_bunto-last-modified-at.rb:30 # Bunto::BuntoSitemap with bunto-last-modified-at correctly adds the modified time to the pages
```

**Comment by suriyaa** [comment]
> Fixed in the :white_check_mark: pull request #8!
